.github/workflows/e2e-nightly-full.yaml

```diff
@@ -1,9 +1,8 @@
-name: Run e2e tests
+name: Run nightly e2e tests
 
 on:
-  push:
```

Comment:
I think this job could run on PR merge and in nightly. No need to disable what works.